src/parser/formatting-evaluator-module.ts

```diff
@@ -91,30 +96,43 @@ export class FormattingEvaluatorModule implements Module {
     const html = this._md.render(comment.content);
     const temp = new JSDOM(html);
     if (temp.window.document.body) {
-      const res = this.classifyTagsWithWordCount(temp.window.document.body);
+      const res = this.classifyTagsWithWordCount(temp.window.document.body, comment.type);
       return { formatting: res };
     } else {
       throw new Error(`Could not create DOM for comment [${comment}]`);
     }
   }
 
-  _countWords(text: string): number {
-    return text.trim().split(/\s+/).length;
+  _countWords(symbols: FormattingEvaluatorConfiguration["multipliers"][0]["symbols"], text: string) {
+    const counts: { [p: string]: { count: number; multiplier: number } } = {};
+    for (const [regex, multiplier] of Object.entries(symbols)) {
+      const match = text.trim().match(new RegExp(regex, "g"));
+      counts[regex] = {
+        count: match?.length || 1,
```

Comment:
if there's no match shouldn't we set to 0?

Reply:
Maybe that is safer yes, will change.

Reply:
I never would select `img` because `a` would always wrap them automatically as a heads up.